.github/workflows/unit-tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-version: [8.1, 8.2, 8.3]
+        php-version: ["8.1", "8.2", "8.3"]
```

Comment:
Can you explain why you changed this? IMHO these are numbers and not strings, so putting them here as floats makes more sense?

Reply:
If you install version `1.10` of anything, YAML will interpret it as `1.1`

Reply:
Please see above.